index.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const models = require('./models')
+const app = express()
+const Op = require('sequelize')
+
+
+app.get('/teams', async (request, response) => {
+  const teams = await models.teams.findAll()
+  response.send(teams)
+})
+
+app.get('/teams/:id', async (request, response) => {
+  const { id } = request.params.id
```

Comment:
So this doesn't work and in fact throws an error. You should be destructuring `request.params` which is an object that has an attribute of `id`. The `request.params.id` should be a string and thus not able to be destructured.

![image](https://user-images.githubusercontent.com/5775716/68711392-8935db80-0567-11ea-953b-5797b1af5b6f.png)